packages/core/src/app/common/utility/isCustomShippingEnabled.ts

```diff
@@ -0,0 +1,9 @@
+import { CheckoutSettings } from '@bigcommerce/checkout-sdk';
+
+export default function isCustomShippingEnabled(
```

Comment:
Can we refactor this and have it generic so we can pass an experiment name and fallback is `true`.